app/src/main/java/com/babylon/wallet/android/presentation/settings/appsettings/AppSettingsViewModel.kt

```diff
@@ -43,6 +48,24 @@ class AppSettingsViewModel @Inject constructor(
                     }
                 }
         }
+        if (BuildConfig.CRASH_REPORTING_ENABLED) {
```

Comment:
Yes it makes sense here to rename it `CRASH_REPORTING_AVAILABLE` since it may be disabled and still available to enable.